src/styles/commonStyles.tsx

```diff
@@ -133,5 +159,99 @@ export const commonStyles = makeStyles()((theme) => {
       color: '#3B4054',
       textAlign: 'center',
     },
+    gridContainer: {
```

Comment:
plz move styles to the according components. And to combine generic common styles and particularly, could you plz fix here hook. Marina did a bit wrong.
https://stackoverflow.com/questions/56537011/creating-styles-from-material-ui-theme-that-can-be-reused-across-components